cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -1,30 +1,35 @@
 import functions_framework
 import pathlib
+import geopandas as gpd
+import pandas as pd
+import numpy as np
 
 from cloudevents.http import CloudEvent
 from google.cloud import firestore
 
 STUDY_AREAS_ID = "study_areas"
+GLOBAL_CRS = "EPSG:4326"
 
 
 # Triggered by the "object finalized" Cloud Storage event type.
 @functions_framework.cloud_event
-def export_model_predictions(cloud_event: CloudEvent) -> dict:
+def export_model_predictions(cloud_event: CloudEvent) -> pd.DataFrame:
     """This function is triggered when a new object is created or an existing
     object is overwritten in the "climateiq-predictions" storage bucket.
 
     Args:
         cloud_event: The CloudEvent representing the storage event.
```

Comment:
I guess it might be helpful to mention that there are some expectations for the name of this event like it should be in the form containing 5 components (`/<prediction_type>/<model_id>/<study_area_name><scenario_id>/<chunk_id>`), right?

Reply:
done